src/utils/aws/ses.py

```diff
@@ -0,0 +1,127 @@
+"""Create DNS email records from SES."""
+# Third-Party Libraries
+import boto3
+
+route53 = boto3.client("route53")
+ses = boto3.client("ses")
+
+
+def list_hosted_zones(names_only: bool = False):
+    """
+    List hosted zones.
+
+    Set names_only to true if only hosted zone names are needed.
+    """
+    if not names_only:
+        return route53.list_hosted_zones()["HostedZones"]
+
+    return [hosted_zone.get("Name") for hosted_zone in list_hosted_zones()]
+
+
+def manage_resource_records(
+    domain_name: str,
+    action: str,
+    verification_token: str,
+    dkim_token_1: str,
+    dkim_token_2: str,
+    dkim_token_3: str,
+):
+    """Manage Route53 Records."""
+    if f"{domain_name}." not in list_hosted_zones(names_only=True):
+        return "Domain's hosted zone does not exist."
+
+    dns_id = "".join(
+        hosted_zone.get("Id")
+        for hosted_zone in list_hosted_zones()
+        if hosted_zone.get("Name") == f"{domain_name}."
+    )
+
+    return route53.change_resource_record_sets(
+        HostedZoneId=dns_id,
+        ChangeBatch={
+            "Comment": "",
+            "Changes": [
+                {
+                    "Action": action,
+                    "ResourceRecordSet": {
+                        "Name": f"_amazonses.{domain_name}",
+                        "Type": "TXT",
+                        "TTL": 300,
+                        "ResourceRecords": [{"Value": f'"{verification_token}"'}],
+                    },
+                },
+                {
+                    "Action": action,
+                    "ResourceRecordSet": {
+                        "Name": f"{dkim_token_1}._domainkey.{domain_name}",
+                        "Type": "CNAME",
+                        "TTL": 300,
+                        "ResourceRecords": [{"Value": f"{dkim_token_1}.amazonses.com"}],
+                    },
+                },
+                {
+                    "Action": action,
+                    "ResourceRecordSet": {
+                        "Name": f"{dkim_token_2}._domainkey.{domain_name}",
+                        "Type": "CNAME",
+                        "TTL": 300,
+                        "ResourceRecords": [{"Value": f"{dkim_token_2}.amazonses.com"}],
+                    },
+                },
+                {
+                    "Action": action,
+                    "ResourceRecordSet": {
+                        "Name": f"{dkim_token_3}._domainkey.{domain_name}",
+                        "Type": "CNAME",
+                        "TTL": 300,
+                        "ResourceRecords": [{"Value": f"{dkim_token_3}.amazonses.com"}],
```

Comment:
I wouldn't create these records. They are for sending only.

Reply:
I also don't see the MX record in this set that is critical for receiving emails. You should just need the TXT and MX record.